### PR TITLE
Feature/650 flyby guidance refactor

### PR DESCRIPTION
**Pull request by thibaudteil**
* **Tickets addressed:** bsk-650
* **Review:** By commit 
* **Merge strategy:** Merge (no squash)  

## Description
Refactor the flybyPointing module to use only Cpp. Refactor adds setters and getters, and performs several small code improvements

## Verification
Test was changed to use setters and getters, but still passes unchanged

## Documentation
None

## Future work
None

### CODE REVIEW

File: src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.cpp
Comment (patkenneally) [inline src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.cpp:91]: :large_orange_diamond: Looks like this change snuck in here from he subsequent commit?

File: src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.h
Comment (patkenneally) [inline src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.h]: :large_orange_diamond: We should class initialize these also. E.g. 
`bool firstRead=true;` and so on

File: src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.cpp
Comment (patkenneally) [inline src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.cpp]: :large_orange_diamond: Is there a preference for declaring and assigning the variable on different lines? I don't know if there's a benefit either way. Just curious.

**Review by patkenneally — APPROVED** [review]
Legend:
:small_red_triangle_down: *Issues* to address before merge
:large_orange_diamond: *Requests* that should not block merge, but should at least be discussed
:large_blue_circle: *Recommendations* that can be ignored if desired

File: src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.cpp
Comment (thibaudteil) [reply src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.cpp:91]: Not sure I understood this one

File: src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.h
Comment (thibaudteil) [reply src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.h]: done

File: src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.cpp
Comment (thibaudteil) [reply src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.cpp]: improved this too

**Pull request merged**
